plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/ax_sweeper.py

```diff
@@ -0,0 +1,324 @@
+                trial_index=batch_of_trials_to_launch[idx].trial_index, raw_data=val,
+            )
+
+    def setup_ax_client(self, arguments: List[str]) -> AxClient:
+        """Method to setup the Ax Client"""
+
+        parameters = []
+        for key, value in self.ax_params.items():
+            parameters.append(OmegaConf.to_container(value, resolve=True))
+            parameters[-1]["name"] = key  # type: ignore
+        commandline_params = self.parse_commandline_args(arguments)
+        for cmd_param in commandline_params:
+            for param in parameters:
+                if param["name"] == cmd_param["name"]:  # type: ignore
+                    for key, value in cmd_param.items():
+                        param[key] = value  # type: ignore
+        ax_client = AxClient(
+            verbose_logging=self.verbose_logging, random_seed=self.random_seed
+        )
+        ax_client.create_experiment(parameters=parameters, **self.experiment)
+
+        return ax_client
+
+    def parse_commandline_args(
+        self, arguments: List[str]
+    ) -> List[Dict[str, Union[ax_types.TParamValue, List[ax_types.TParamValue]]]]:
+        """Method to parse the command line arguments and convert them into Ax parameters"""
+
+        def _is_int(string_inp: str) -> bool:
+            """Method to check if the given string input can be parsed as integer"""
+            try:
+                int(string_inp)
+                return True
+            except ValueError:
+                return False
+
+        def _is_float(string_inp: str) -> bool:
+            """Method to check if the given string input can be parsed as a float"""
+            try:
+                float(string_inp)
+                return True
+            except ValueError:
+                return False
+
+        parameters = []
+        for arg in arguments:
+            key, value = arg.split("=")
+            if "," in value:
+                # This is a Choice Parameter.
+
+                value_choices = [x for x in value.split(",")]
+                if _is_float(value_choices[0]):
+                    parameter_type = ParameterType.FLOAT
+                    value_choices = [float(x) for x in value_choices]  # type: ignore
+                else:
+                    parameter_type = ParameterType.STRING
+                parameters.append(
+                    {
+                        "name": key,
+                        "type": "choice",
+                        "values": value_choices,
+                        "parameter_type": parameter_type,
+                    }
+                )
+            elif ":" in value:
+                # This is a Range Parameter.
+                range_start, range_end = value.split(":")
+                if _is_int(range_start) and _is_int(range_end):
+                    range_start = int(range_start)  # type: ignore
+                    range_end = int(range_end)  # type: ignore
+                    param_type = ParameterType.INT
```

Comment:
If there is a conflict between the command line and the config what is taking priority?

Reply:
command line